x-pack/plugins/alerting/server/rules_client/lib/validate_attributes.ts

```diff
@@ -92,7 +92,7 @@ export const iterateFilterKureyNode = ({
     } else if (ast.type === 'literal' && ast.value && typeof ast.value === 'string') {
       const key = ast.value.replace('.attributes', '');
       const mappingKey = 'properties.' + key.split('.').join('.properties.');
-      const field = get(mappings, mappingKey);
+      const field = get(alertMappings, mappingKey);
       if (field != null && field.type === 'nested') {
         localNestedKeys = ast.value;
       }
```

Comment:
It looks like this code never worked. `field` was always `undefined` because the `mappingKey` starts with `properties.` but according to the `mappings.json` structure, the key should start with `alert.properties.`.